### PR TITLE
chore(tests): Add policy based tests

### PR DESCRIPTION
**Pull request by simar7**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,9 +1,12 @@
...
 
-go 1.20
+go 1.21
+
+toolchain go1.21.4
 
 require (
-	github.com/aquasecurity/defsec v0.93.2-0.20240104002958-968b8f115bc0
+	github.com/aquasecurity/defsec v0.94.1
+	github.com/aquasecurity/trivy v0.0.0-00010101000000-000000000000
```
Comment (simar7) [inline go.mod:9]: update when https://github.com/aquasecurity/trivy/pull/6005 is merged

**Comment by nikpivkin** [comment]
Are these tests really supposed to be here ? I don't see the `trivy-policies` import, but there is defsec and trivy.
[This](https://github.com/aquasecurity/trivy-policies/pull/70/files#diff-ccd951d0ab2f75ba0337dcb73aad6b709f40d5a004eb2ae01f40ed467540f578R70) is the only place where the checks from this repository are used .

**Comment by simar7** [comment]
> Are these tests really supposed to be here ? I don't see the `trivy-policies` import, but there is defsec and trivy.
> [This](https://github.com/aquasecurity/trivy-policies/pull/70/files#diff-ccd951d0ab2f75ba0337dcb73aad6b709f40d5a004eb2ae01f40ed467540f578R70) is the only place where the checks from this repository are used .

Do you mean these tests shouldn't be here? The imports are from trivy and defsec both because we haven't merged the defsec code into trivy just yet.

**Review by nikpivkin — APPROVED** [review]
LGTM

**Pull request merged**